rsts/community/contribute.rst

```diff
@@ -284,68 +284,373 @@ To understand how the below components interact with each other, refer to :ref:`
+   # 📂 The Minio API is hosted on localhost:30002. Use http://localhost:30080/minio/login for Minio console
+
+   # Step2: Export FLYTECTL_CONFIG as the previous log indicated.
+   FLYTECTL_CONFIG=/home/ubuntu/.flyte/config-sandbox.yaml
+
+   # Step3: The kubeconfig will be automatically copied to the user's main kubeconfig (default is `/.kube/config`) with "flyte-sandbox" as the context name.
+   # Check that we can access the K3s cluster. Verify that Postgres and Minio are running.
+   kubectl get pod -n flyte
+   # NAME                                                  READY   STATUS    RESTARTS   AGE
+   # flyte-sandbox-docker-registry-85745c899d-dns8q        1/1     Running   0          5m
+   # flyte-sandbox-kubernetes-dashboard-6757db879c-wl4wd   1/1     Running   0          5m
+   # flyte-sandbox-proxy-d95874857-2wc5n                   1/1     Running   0          5m
+   # flyte-sandbox-minio-645c8ddf7c-sp6cc                  1/1     Running   0          5m
+   # flyte-sandbox-postgresql-0                            1/1     Running   0          5m
+
+**3. [Optional] Access the Minio console via** http://localhost:30080/minio/login.
+
+
+| The default Username is ``minio`` and the default Password is ``miniostorage``.  
+| You might need to look at ``input.pb``, ``output.pb`` or ``deck.html``, etc in Minio when you are developing.
+
+**4. Run all backends(flyteidl, flyteadmin, flyteplugins, flytepropeller, console) in a single binary.**
+
+The `Flyte repository <https://github.com/flyteorg/flyte>`__ includes Go code
+that integrates all backend components (flyteidl, flyteadmin, flyteplugins, flytepropeller, console, etc) into a single binary.
+
+.. code:: shell
+
+   # Step1: Download flyte repo
+   git clone https://github.com/flyteorg/flyte.git
+   cd flyte
+
+   # Step2: Build a single binary that bundles all the backends (flyteidl, flyteadmin, flyteplugins, flytepropeller) and HTTP Server.
+   # The versions of flyteidl, flyteadmin, flyteplugins, and flytepropeller used to build the single binary are defined in `go.mod`.
+   sudo apt-get -y install jq # You may need to install jq
+   go mod tidy
+   sudo -E env "PATH=$PATH" make compile
+
+   # Step3: Edit the config file: ./flyte-single-binary-local.yaml.
+   # Replace occurrences of $HOME with the actual path of your home directory.
+   sed -i "s|\$HOME|${HOME}|g" ./flyte-single-binary-local.yaml
+
+   # Step4: Running the single binary.
+   # The POD_NAMESPACE environment variable is necessary for the webhook to function correctly. 
+   # You may encounter an error due to `ERROR: duplicate key value violates unique constraint`. Run the command again will solve the problem.
+   POD_NAMESPACE=flyte flyte start --config flyte-single-binary-local.yaml
+   # All logs from flyteadmin, flyteplugins, flytepropeller, etc. will appear in the terminal.
+
+**5. [Optional] Access the Flyte UI at** http://localhost:30080/console.
```

Comment:
maybe have access to Flyte UI, minio, postgress, k8s, endpoints in a single section rather than separate minio and flyte UI access?